server/routes/todo.js

```diff
@@ -55,37 +63,40 @@ export default function (router) {
             return res.status(400).send("Name was null.");
         }
 
+        // Insert command
+        const insertCommand = "INSERT INTO todos (name, completed) VALUES ($1, $2)";
+
         // Do database insertion.
-        pool.query('INSERT INTO todos (name, completed) VALUES ($1, $2)', [name, false], (err, result) => {
-            if (err) {
-                console.log("Error inserting todo: ", err);
-                return res.status(400).send("Error creating todo");
-            } else {
-                console.log("Successfully added a new todo!", result);
-                return res.status(200).send("Created todo");
-            }
-        })
-        // pool.query('INSERT INTO todos (name, completed) VALUES ($1, $2)', [name, completed], (error, results) => {
-        //   if (error) {
-        //     throw error
-        //   }
-        //   response.status(201).send(`User added with ID: ${result.insertId}`)
+        try {
+            const result = await pool.query(insertCommand, [name, false]);
+            console.log("Successfully added a new todo!", result);
+            return res.status(200).send("Created todo");
+        } catch (error) {
+            console.log(error);
+            return res.status(400).send("Error creating todo");
+        }
     });
 
     /**
      * Deletes a todo from the database by ID.
      */
     router.delete(`/todos/:id`, async (req, res) => {
+
         const id = Number(req.params.id);
 
         if (!isNaN(id) && id > 0) {
-            pool.query('DELETE FROM todos WHERE id = $1;', [id], (err, result) => {
-                if (err) {
-                    console.log("Error deleting todo: ", err);
-                } else {
-                    console.log("Sucessfully deleted todo!", result);
-                }
-            })
+
+            const deleteCommand = "DELETE FROM todos WHERE id = $1;";
+
+            try {
+                const result = await pool.query(deleteCommand, [id]);
+                console.log("Sucessfully deleted todo!", result);
```

Comment:
You need to send a response here. Something akin to `res.status(200)...` like you did above.

Reply:
All routes now send appropriate responses